sr_data_visualization/launch/data_visualizer_testing.launch

```diff
@@ -0,0 +1,41 @@
+<launch>
+
+
+  <arg name="hand_id" default="rh"/>
```

Comment:
Not sure what is the purpose of this launch file?

Reply:
It allowed us to visualise the meshes of the hand without starting a full simulation of it. Also at least some of it is needed to run the biotac visualiser widgets (if a real hand isn't connected). I'll find out what's necessary for just the biotac visualisers and take the rest out.